website/versioned_docs/version-0.60/typescript.md

```diff
@@ -33,7 +33,7 @@ ignite new MyTSProject
 1. Add TypeScript and the types for React Native and Jest to your project.
 
 ```sh
-yarn add typescript @types/jest @types/react @types/react-native @types/react-test-renderer
```

Comment:
The file correct for changes is: 'docs/typescript.md'

Reply:
got it ..